### DISCUSSION TITLE
Apator WM-bus-08 flow

### DISCUSSION THREAD

**Discussion by DybaMati** [wmbusmeters/wmbusmeters#506] [General]
Hello,
Does anyone know if the apator 08 overlay has information about the current m3/h or l/h water flow ??

**Comment by weetmuts** [comment]
Hi @DybaMati ! I think this was answered here: https://github.com/weetmuts/wmbusmeters/issues/507
So I am closing this discussion. Please open an issue/discussion if you want to assist in reverse engineering the apator08 more.
I cannot really help since I do not have that meter myself.
